docs/utils.rst

```diff
@@ -23,3 +23,27 @@ so you can schedule, for instance, an hourly cronjob such as
 .. code-block:: bash
 
     $ python manage.py clean_duplicate_history -m 60 --auto
+
+
+clean_old_history
+-----------------------
+
+After a While your Historical Records start to get a little bit to big, and maybe you don't
+need to save history for more than x amount of days,
+
+If you find yourself with a lot of olf history you can schedule the
+``clean_old_history`` command
+
+.. code-block:: bash
+
+    $ python manage.py clean_old_history --auto
+
+You can use ``--auto`` to clean up duplicates for every model
+with ``HistoricalRecords`` or enumerate specific models as args.
```

Comment:
duplicates? I thought this was just old?

Reply:
Done!